spec/migrations/20160322205357_remove_host_id_from_switch_spec.rb

```diff
@@ -0,0 +1,42 @@
+require_migration
+
+describe RemoveHostIdFromSwitch do
+  before(:each) do
+    @host = host_stub.create!
+    @switch1 = switch_stub.create!(:hosts => [@host])
+    @switch2 = switch_stub.create!(:hosts => [@host])
+  end
+
+  migration_context :up do
+    let(:host_stub) { migration_stub(:Host) }
+    let(:switch_stub) { migration_stub(:Switch) }
+    it 'Remove column switches.host_id' do
+      expect(@switch1).to respond_to('host_id')
+      expect(@host.switches.length).to eq 2
+
+      migrate
+
+      @host.reload
+      expect(@host.switches.length).to eq 2
+      @switch1.reload
+      expect(@switch1).not_to respond_to('host_id')
+    end
+  end
```

Comment:
I believe the up of this migration is only a schema change so I think we can skip testing up migration.